tool/settings.re

```diff
@@ -121,13 +122,20 @@ impl Settings {
                 Option::None => default,
             }
         };
+        let get_string = fn(key: string, default: string) -> string {
+            match map.get(key) {
+                Option::Some(val) => val,
+                Option::None => default,
+            }
+        };
         Settings {
             ui_scale: get_float("ui_scale", 2.),
             show_character_stats: get_bool("show_character_stats", false),
             show_game_stats: get_bool("show_game_stats", false),
             minimap_enabled: get_bool("minimap_enabled", true),
             minimap_size: get_float("minimap_size", 0.35),
             minimap_alpha: get_float("minimap_alpha", 0.4),
+            minimap_position: get_string("minimap_position", "TopRight"),
```

Comment:
Do you think it makes more sense to have the default be bottom-right or top-right?